recipes/ipydatetime/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "ipydatetime" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9b083e08efef140c29091fe114ea2921f88989e56f0b5eecdde9ac0157bb0b10
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -c '__import__("os").unlink("pyproject.toml")'
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - jupyter-packaging
+    - python >=3.6
+    - pip
+  run:
+    - ipywidgets >=7.0.0
+    - jupyterlab >=2
+    - nodejs
+    - python >=3.6
+    - tzlocal
+
+test:
+  commands:
+    - pip check
+    - jupyter labextension check @jupyterlab/jupyter-widget-datetime; echo $?
```

Comment:
try a `jupyter labextension list` as well. Though i don't see the tarball getting copied in the log... will make a suggestion above.

Reply:
also the fully-qualified name is `jupyter-widget-datetime`, and might not show up as "working" unless built, so perhaps:
```
- jupyter lab build
- jupyter labextension list
- jupyter labextension check jupyter-widget-datetime; echo $?
```